person.rb

```diff
@@ -9,9 +9,14 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
-  [ ] As the reviewer required, please pass the book, date, and person as arguments for this method. Then, you need to create a rental as the reviewer made for you. Add that rental to @rentals which belongs to a person. So, you need to give arguments (book, date, self) and create a new rental, pass that rental with `@rentals << rental`. 
- [ ] The same thing applies to books, in the add_rental method please add arguments(person, date, self) to create the rental and pass it to the rentals of the book.